src/nadzoring/dns_lookup/reverse.py

```
@@ -132,11 +130,11 @@ def reverse_dns(
     except dns.exception.Timeout:
         result["error"] = "Query timeout"
         logger.debug("Reverse DNS timeout for %s", ip_address)
-    except ValueError as exc:
-        result["error"] = f"Invalid IP address: {exc}"
+    except ValueError:
+        result["error"] = "Invalid IP address"
         logger.debug("Invalid IP for reverse lookup: %s", ip_address)
-    except Exception as exc:
-        result["error"] = str(exc)
-        logger.debug("Reverse DNS failed for %s: %s", ip_address, exc)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)